### PR TITLE
[15.0][FIX] chained_swapper: Change SavepointCase to TransactionCase

### PR DESCRIPTION
**Pull request by victoralmau**
Changes done:
- [x] Change SavepointCase to TransactionCase
- [x] Change load_lang method to _activate_lang to prevent warning log

@Tecnativa

### CODE REVIEW

File: chained_swapper/tests/test_chained_swapper.py [hunk 1]
```diff
@@ -10,11 +10,11 @@
 from ..hooks import uninstall_hook
 
 
-class TestChainedSwapper(common.SavepointCase):
+class TestChainedSwapper(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super(TestChainedSwapper, cls).setUpClass()
-        cls.env["res.lang"].load_lang("es_ES")
+        cls.env["res.lang"]._activate_lang("es_ES")
```
Comment (pedrobaeza) [inline chained_swapper/tests/test_chained_swapper.py:17]: You can directly remove this line, as English is always preloaded.

Comment (victoralmau) [reply chained_swapper/tests/test_chained_swapper.py:17]: IMO, this is necessary to test what the tests do.

Comment (pedrobaeza) [reply chained_swapper/tests/test_chained_swapper.py:17]: Oh, sorry, I saw badly that it loads English.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**